CMakeLists.txt

```diff
@@ -3,10 +3,12 @@ project(rtrlib)
 cmake_minimum_required(VERSION 2.6)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
-
+set(NOSSH false)
```

Comment:
btw. Travis fails not because of my changes, its ```cppcheck``` which fails consistently. So maybe we should disabled it for now in Travis?